src/Interpreters/Context.cpp

```diff
@@ -367,6 +368,9 @@ struct ContextSharedPart : boost::noncopyable
     std::atomic_size_t max_view_num_to_warn = 10000lu;
     std::atomic_size_t max_dictionary_num_to_warn = 1000lu;
     std::atomic_size_t max_part_num_to_warn = 100000lu;
+    /// Only for system.server_settings, actually value stored in reloader itself
```

Comment:
Why do we need to store it in the ContextReloader if we have it in Context?

Reply:
Because ConfigReloader doesn't know anything about Context and vise-versa, and I don't want to couple them...